src/libmongoc/src/mongoc/mongoc-bulkwrite.c

```diff
@@ -1438,7 +1438,7 @@ _bulkwritereturn_apply_result (mongoc_bulkwritereturn_t *self,
    return true;
 }
 
-BSON_EXPORT (void)
+MONGOC_EXPORT (void)
```

Comment:
```suggestion
void
```

Reply:
I expect `MONGOC_EXPORT` / `BSON_EXPORT` is only needed in public headers, and is harmless (though unnecessary) in source files.

The only other occurrence is [mongoc_client_encryption_new](https://github.com/mongodb/mongo-c-driver/blob/2c4dd7581beb65fcf97b44b68f36e9ba7152733c/src/libmongoc/src/mongoc/mongoc-client-side-encryption.c#L893), which I expect can be removed too.

Reply:
Noted. I'll remove this here and in the new APIs introduced in https://github.com/mongodb/mongo-c-driver/pull/1805.